Specification/Instructions/liquid_handle.md

```diff
@@ -31,7 +31,8 @@ Transports within the same locations use the same consumables (i.e. tips in the
           "mode_params": Option<{
             "liquid_class": Option<Enum(
               "air",
-              "default"
+              "default",
```

Comment:
there's mention of `Protein` in the ASC

Reply:
Protein (which should've been `protein_buffer`) is available only in txappy (for tempest). Deleting Protein from ASC.